src/list/actions.js

```diff
@@ -331,3 +344,47 @@ export function openFAQ() {
     type: OPEN_FAQ,
   };
 }
+
+export function openSyncPrefs() {
+  window.open(SYNC_PREFS_URL, "_blank");
+  return { type: OPEN_SYNC_PREFS };
+}
+
+export function openAccount() {
+  window.open(ACCOUNT_URL, "_blank");
+  return { type: OPEN_ACCOUNT };
+}
+
+export function openConnectDevice() {
+  window.open(CONNECT_DEVICE_URL, "_blank");
```

Comment:
this is to be an in-content dialog; addressed in issue #TBD 😄 (though there are [designs](https://mozilla.invisionapp.com/share/ENOL84959AK#/screens/342439032))